<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 22, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CameraVideo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(87, 96, 100,127);
        border-style: outset;
        border-width: 1px;
        border-radius: 5px;
        border-color: rgba(255, 255, 255, 0);
        padding: 4px;
        color: rgb(255, 243, 247);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(94, 99, 97);
        border-style: outset;
        border-width:0;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: #fbfbfb;
        font-size: 13px;
        font-weight: 900;}
        
        QPushButton:hover{background-color:  rgb(94, 99, 97);
        border-style: outset;
        border-width: 0px;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: #565050;
        font-size: 13px;
        font-weight: 700;}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndExamButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(94, 99, 97);
        border-style: outset;
        border-width:0;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(0, 0, 127);
        font-size: 13px;
        font-weight: 900;}
        
        QPushButton:hover{background-color:  rgb(94, 99, 97);
        border-style: outset;
        border-width: 0px;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(46, 63, 127);
        font-size: 13px;
        font-weight: 700;}</string>
    </property>
    <property name="text">
     <string>End Exam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TimerNote">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgba(94, 99, 97,0);
        border-style: outset;
        border-width:0;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(70, 211, 0);
        font-size: 13px;
        font-weight: 900;}
        
        QPushButton:hover{background-color:  rgb(94, 99, 97);
        border-style: outset;
        border-width: 0px;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(46, 63, 127);
        font-size: 13px;
        font-weight: 700;}</string>
    </property>
    <property name="text">
     <string>0:0:0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LightNote">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgba(94, 99, 97,0);
        border-style: outset;
        border-width:0;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(70, 211, 0);
        font-size: 13px;
        font-weight: 900;}
        
        QPushButton:hover{background-color:  rgb(94, 99, 97);
        border-style: outset;
        border-width: 0px;
        border-radius: 5px;
        border-color: #e8e8e8;
        color: rgb(46, 63, 127);
        font-size: 13px;
        font-weight: 700;}</string>
    </property>
    <property name="text">
     <string>Increase Light</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
